building/target_platforms/target_platforms_for_consumers.adoc

```diff
@@ -0,0 +1,212 @@
+
+* https://github.com/jbosstools/jbosstools-target-platforms[JBoss Tools & Red Hat JBoss Developer Studio Target Platforms]
+* https://github.com/jbosstools/jbosstools-discovery[JBoss Tools Central Target Platforms]
+* https://github.com/jbosstools/jbosstools-integration-stack[Integration Stack Target Platforms]
+
+JBoss Tools' `target platform update sites` are located here, in both unpacked and packed (zipped) format. They do *not* contain sources or SDK features.
+
+* http://download.jboss.org/jbosstools/targetplatforms/ (SNAPSHOTS and milestones)
+* http://download.jboss.org/jbosstools/static/targetplatforms/ (released versions)
+
+## Exploring the JBoss Tools target platform repository
+
+Inside the JBoss Tools Target Platform repository, the two top level folders are `jbdevstudio` and `jbosstools`.  Both of these folders contain two sub-folders: `multiple` and `unified`. 
+
+### Multiple vs unified target platforms
+
+**Multiple** target platform definition files reference several different update sites, most of which are mirrors located on http://download.jboss.org/jbosstools/. Use this .target file in Eclipse, since it can include sources when built with the proper flags.
+
+**Unified** target platform definition files are generated from the multiple ones, and are then used to produce the target platform update sites. As this file references only a single mirror, it is faster to use in Eclipse but cannot include sources, since it points at the generated target platform update site, which does not include sources.
+
+### Versioning rules
+
+JBoss Tools has a semi-intuitive versioning scheme for our target platforms. 
+
+The first 3 digits are the version of Eclipse included in the target platform, such as 4.41 (Luna SR1) or 4.50 (Mars R).
+
+The next digit signifies the sequence of target platforms, starting with 0. Should there be multiple JBoss Tools releases on the same Eclipse version, this allows us to have different target platforms using the same baseline, such as 4.42.0, 4.42.1, 4.42.2.
+
+After that, the milestone of JBoss Tools that uses the target platform is suffixed, which also defines the quality/stability of the target platform (Alpha1, Alpha2, Beta1, Beta2, CR1, Final). 
+
+Finally, the target platform version will have a `-SNAPSHOT` suffix if it has not yet been released to Nexus. Once released, the `-SNAPSHOT` suffix is removed and the target platform update site is moved to http://download.jboss.org/jbosstools/static/targetplatforms/. 
+
+### Choosing the appropriate target platform.
+
+Use the latest target platform for the version of Eclipse your development environment requires. For example, if you are building code for Mars, use a https://github.com/jbosstools/jbosstools-target-platforms/tree/4.50.x[4.50.x] target platform. If building for Luna SR1 or SR2, use a https://github.com/jbosstools/jbosstools-target-platforms/tree/4.41.x[4.41.x] or 4.42.x target platform.
```

Comment:
Should add something like "normally each project uses a parent pom that will default to the right target platform but in case you need to update/test a different target platform use this".